cves/CVE-2016-6817.yml

```diff
@@ -143,8 +143,8 @@ interesting_commits:
     section by explaining what happened between the VCCs and the fix.
   answer:
   commits:
-    - commit:
-      note:
+    - commit: 0cb56058ee7f237764cc4435a429d366e6b7a128
+      note: This commit was about writing unit tests for the HTTP2 pings. I found it interesting because it was added out of nowhere, which made me think why they decided to add unit tests for something out of the blue. Also, I noticed that the HTTP2 header parser also didn't have unit tests, until the vulnerability was found, which I thought was interesting.
```

Comment:
`Also, I noticed that the HTTP2 header parser also didn't have unit tests, until the vulnerability was found, which I thought was interesting.`
Avoid using 'also' twice. 
This sentence feels like it runs on a bit long. Try rewording into 2 sentences.